.buildkite/hooks/post-command

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -eu
+
+DEPENDENT_PROJECT='cogctl'
+TRIGGER_PIPELINE_SLUG=${DEPENDENT_PROJECT}
+
+if git ls-remote --exit-code --heads https://github.com/operable/$DEPENDENT_PROJECT refs/heads/${BUILDKITE_BRANCH} > /dev/null 2>&1
+then
+    TRIGGER_BRANCH=${BUILDKITE_BRANCH}
+else
+    TRIGGER_BRANCH='master'
+fi
+
+TRIGGER_MESSAGE=":cogops: Dependent Build Triggered from ${BUILDKITE_BUILD_URL} :cogops:"
+
+curl \
+  "https://api.buildkite.com/v2/organizations/${BUILDKITE_ORGANIZATION_SLUG}/pipelines/${TRIGGER_PIPELINE_SLUG}/builds" \
+  -X POST \
+  --fail \
+  -H "Authorization: Bearer ${BUILDKITE_API_ACCESS_TOKEN}" \
+  -d "{
+    \"commit\": \"HEAD\",
+    \"branch\": \"${TRIGGER_BRANCH}\",
+    \"message\": \"${TRIGGER_MESSAGE}\",
+    \"env\": {
+      \"COG_BRANCH\": \"${BUILDKITE_BRANCH}\"
+    }
+  }"
```

Comment:
@vanstee I've added a json header, but absent any specific `Content-Type` header, `curl` will send `Content-Type application/x-www-form-urlencoded` when you use the `-d`/`--data` options, so I'm guessing that Buildkite accepts that as well. I think we do something similar for triggers, too.